.github/workflows/semgrep.yml

```diff
@@ -12,7 +11,7 @@ name: Semgrep config
 jobs:
   semgrep:
     name: semgrep/ci
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
Can we pin to whatever the latest version is, to make it reproducible?

Reply:
updated the PR